flowey/flowey_lib_hvlite/src/download_openhcl_kernel_package.rs

```diff
@@ -139,10 +139,6 @@ impl FlowNode for Node {
                     if cfg!(unix) {
                         #[cfg(unix)]
                         {
-                            // HACK: recursively chmod all the files, otherwise they all have 000 access.
```

Comment:
All releases are now tar.gz so this is no longer needed to work around .zip files having zero permissions.